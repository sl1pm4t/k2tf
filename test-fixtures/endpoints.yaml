apiVersion: v1
kind: Endpoints
metadata:
  annotations:
    alpha.istio.io/kubernetes-serviceaccounts: default
  name: backend
  namespace: default
subsets:
  - addresses:
      - ip: 10.128.0.17
    ports:
      - name: http-backend
        port: 8080
        protocol: TCP
  - notReadyAddresses:
      - ip: 10.128.0.11
    ports:
      - name: http-backend
        protocol: TCP
        port: 8080