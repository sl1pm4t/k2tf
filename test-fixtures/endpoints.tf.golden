resource "kubernetes_endpoints" "backend" {
  metadata {
    name      = "backend"
    namespace = "default"
    annotations {
      "alpha.istio.io/kubernetes-serviceaccounts" = "default"
    }
  }
  subset {
    address {
      ip = "10.128.0.17"
    }
    port {
      name     = "http-backend"
      port     = 8080
      protocol = "TCP"
    }
  }
  subset {
    not_ready_address {
      ip = "10.128.0.11"
    }
    port {
      name     = "http-backend"
      port     = 8080
      protocol = "TCP"
    }
  }
}
